adapters/appnexus/appnexus.go

```diff
@@ -284,15 +284,18 @@ func preprocess(imp *openrtb2.Imp, defaultDisplayManagerVer string) (string, boo
 
 	// Accept legacy Appnexus parameters if we don't have modern ones
 	// Don't worry if both is set as validation rules should prevent, and this is temporary anyway.
-	if appnexusExt.PlacementId == 0 && appnexusExt.LegacyPlacementId != 0 {
-		appnexusExt.PlacementId = appnexusExt.LegacyPlacementId
+	if appnexusExt.PlacementId == 0 && appnexusExt.DeprecatedPlacementId != 0 {
+		appnexusExt.PlacementId = appnexusExt.DeprecatedPlacementId
 	}
 	if appnexusExt.InvCode == "" && appnexusExt.LegacyInvCode != "" {
 		appnexusExt.InvCode = appnexusExt.LegacyInvCode
 	}
 	if appnexusExt.TrafficSourceCode == "" && appnexusExt.LegacyTrafficSourceCode != "" {
 		appnexusExt.TrafficSourceCode = appnexusExt.LegacyTrafficSourceCode
 	}
+	if appnexusExt.DeprecatedUsePaymentRule != nil && *appnexusExt.DeprecatedUsePaymentRule {
+		appnexusExt.UsePmtRule = appnexusExt.DeprecatedUsePaymentRule
+	}
```

Comment:
If both the new and deprecated payment rule are provided, this code will use the deprecated one. This is the opposite behavior as the PlacementID.

Reply:
After discussion with the team, I have updated to precede `use_pmt_rule` over `usePaymentRule`. Also changed appnexus.json to reflect this change. Here is the change - https://github.com/prebid/prebid-server/pull/2617/commits/d4fff68d92d3d2dfd1042ea36c45c70e18c7f01b